README.md

```diff
@@ -31,6 +31,9 @@ Proxy for handling the cache for pages within the legacy CMS
 | ENABLE_PUBLISH_EXPIRY_OFFSET | false                  | Determines if publish expiry offset is used which enables a shorter cache time for recently published content.                       |
 | PUBLISH_EXPIRY_OFFSET        | 3m                     | Period of time after a release in which the proxy needs to return a short value for the `max-age` directive (`time.Duration` format) |
 
+### Secrets are Auto-Deployed
```

Comment:
Is this correct? This is new behaviour that I'm unaware of so just checking.

Reply:
Similar to the other PR for dp-legacy-data-cache (https://github.com/ONSdigital/dp-legacy-cache-api/pull/29/files#r1515951687), can I suggest changing the heading and first sentence to make it clearer that this new functionality is non-functional with the current vault/nomad.